src/main/java/com/mycompany/myapp/service/CSV/ParsingContainer.java

```diff
@@ -0,0 +1,93 @@
+package com.mycompany.myapp.service.CSV;
+
+import com.mycompany.myapp.domain.Pool;
+
+import java.util.LinkedList;
+
+public class ParsingContainer {
+    private String poolId;
+    private String displayName;
+    private Integer maximumCount;
+    private Boolean enabled;
+    private String description;
+
+    public ParsingContainer() {
+
+    }
+
+    //    ugly but CSV file format is that ugly
+    public void setPool_id(String pool_id) {
+        this.poolId = pool_id;
+    }
+
+    public void setPoolId(String pool_id) {
+        this.poolId = pool_id;
+    }
+
+    public void setDisplayName(String displayName) {
+        this.displayName = displayName;
+    }
+
+    public void setMaximumCount(Integer maximumCount) {
+        this.maximumCount = maximumCount;
+    }
+
+    public void setEnabled(Boolean enabled) {
+        this.enabled = enabled;
+    }
+
+    public void setDescription(String description) {
+        this.description = description;
+    }
+
+    //    ugly but CSV file format is that ugly
+    public String getPool_id() {
+        return poolId;
```

Comment:
This getter, if necessary, can call another getter if they return the same field